src/extension.ts

```diff
@@ -153,6 +153,16 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
             }
         });
     }
+    if (datasetProvider || ussFileProvider || jobsProvider) {
```

Comment:
Not worth adding this so close to v1.5 but the three trees have a superclass ZoweTreeProvider so you could have ZoweTreeProvider.
No action required.